.github/workflows/fpga-subsystem.yml

```diff
@@ -286,6 +286,7 @@ jobs:
               --target-dir-remap="${TEST_BIN}/target"
               --workspace-remap=.
               -E 'package(caliptra-hw-model) and test(tests::test_execution)'
+              -E 'package(caliptra-drivers) and test(test_dma_aes)'
```

Comment:
I think line 292 might need `--features fpga_subsystem` for this test to appear in the list. 

I don't think this will impact the actual tests being run, since the test binaries were built earlier

Reply:
It looks like `--features` cannot be specified if `--binaries-metadata` is also:

```
error: the argument '--binaries-metadata <PATH>' cannot be used with:
...
--features
```